network/ipam/concepts.mdx

```diff
@@ -0,0 +1,54 @@
+---
+meta:
+  title: IPAM - Concepts
+  description: This page explains all the concepts related to IPAM
+content:
+  h1: IPAM - Concepts
+  paragraph: This page explains all the concepts related to IPAM
+tags: ipam ip-address-management internet-protocol private-ip public-ip flexible-ip list-ips book-ip
+dates:
+  validation: 2024-05-30
+categories:
+  - networks
+---
+
+## IPAM
+
+**IP** **A**ddress **M**anager (IPAM) is Scaleway's tool for planning, tracking and managing the IP address space of Scaleway products. It acts as a single source of truth for the IP addresses of Scaleway resources. Managed products, such as Databases and Load Balancer, are fully integrated into IPAM, and Private Networks' [DHCP server](/network/vpc/concepts/#dhcp) uses IPAM to assign private, static IP addresses to attached resources.
+
+The public rollout of IPAM functionality is being carried out gradually: IPAM is currently in [public beta](https://www.scaleway.com/en/betas/).
+
+- In the [console](https://console.scaleway.com/ipam/), you can currently use IPAM to view, tag and filter all your [private IP addresses](#private-ip-address)
+- With the [API](https://www.scaleway.com/en/developers/api/ipam/), you can additionally [reserve](https://www.scaleway.com/en/developers/api/ipam/#path-ips-book-a-new-ip) an available IP from a Private Network's CIDR block, or [release](https://www.scaleway.com/en/developers/api/ipam/#path-ips-release-an-ip) a previously-reserved address that you no longer want.
+
+More functionality is planned for the future, including:
+
+- Reserve/release private IP addresses in the console
+- Select a reserved private IP address to use when attaching a resource to a Private Network
+- Management of public IP addresses
+
+## IPv4
+
+Internet Protocol Version 4 is the standard protocol used for IP addresses, and routes most internet traffic as of today. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g. `72.16.254.1`.
```

Comment:
The IP `72.16.254.1` belongs to the IP space of Microsoft. 
Any particular reason for this example?

Reply:
This has been in the doc for ages, I copied/pasted it into IPAM from elsewhere in the network doc. No particular reason as far as I know, we can change it to something SCW-ish